plugins/auth/index.js

```diff
@@ -178,12 +178,15 @@ exports.register = (server, options, next) => {
                                         const description =
                                             `The default collection for ${user.username}`;
 
-                                        collectionFactory.get({
-                                            userId: user.id,
-                                            type: 'default'
+                                        collectionFactory.list({
+                                            userId: user.id
```

Comment:
need to have `params` key when using `list` like https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/commands/listTags.js#L28-L34

Reply:
should also still be able to list by `type`

Reply:
@Code-Beast Please take a look.